src/base.cls.php

```diff
@@ -514,6 +515,7 @@ class Base extends Root
 
 		// Crawler
 		self::O_CRAWLER => false,
+		self::O_CRAWLER_SCHEDULE_TIME => '00:00-23:59',
```

Comment:
By default it should be empty, which means allow all time.

Reply:
done